Mlem/Views/Tabs/Posts/Components/Post/Components/Community View/Community View.swift

```diff
@@ -478,6 +478,12 @@ struct CommunityView: View
                 account: account
             )
         } catch APIClientError.networking {
+            // TODO: we're seeing a number of SSL related errors on some instances while loading pages from the feed
+            // while we investigate the reasons we will only show this error if the user would otherwise be left with an empty feed
```

Comment:
Turn these into a bug issue, maybe?

Reply:
👍 I'll raise some for the outstanding ones when I get back to a computer

Reply:
What do you guys think, safe to merge in the current state?

Reply:
I've added three issues for these:

#179 
#180 
#181 

The fixes included in this PR were put up as I thought we'd have wanted to address them prior to rolling the build out for public testing.

The frequent alerts will likely be annoying to users, and not rendering post bodies for web/image posts seems like a blocking issue.

If we're not planning to roll out a fix in the short term these could be left out to see if we reach a better resolution, however having them in place at least fixes the immediate problems from a UX perspective 🙏

Reply:
I've dropped one of the commits that was on this branch as it is now resolved by #193